config/config.json

```diff
@@ -6,7 +6,7 @@
 				],
 	"parserobots":false,
 	"debug":true,
-	"output":"sitemap.xml",
+	"output":"/home/python-sitemap/output/sitemap.xml",
```

Comment:
This change break the « normal » usage aka without Docker. Can you make it transparent?